### DISCUSSION TITLE
LF5 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#144] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF5.html

Author: @AbbyANoble  
Reviewer: @kathypinzon

**Comment by AbbyANoble** [comment]
I sent in a pull request for this one @kathypinzon!

**Comment by kathypinzon** [comment]
remark 3.5.1 can you link to the previous activity?

**Comment by kathypinzon** [comment]
3.5.2 typo - "additional"

**Comment by kathypinzon** [comment]
3.5.5 b - to represent "the" cost

**Comment by kathypinzon** [comment]
Is it possible to add another example that doesn't deal with money?  maybe something with temperature since we use that in another part of the book (in inverses section)
